src/components/Notifications/Table.tsx

```diff
@@ -0,0 +1,346 @@
+import * as React from 'react';
+import {
+    expandable,
+    ICell,
+    IRow,
+    IRowData,
+    RowWrapperProps,
+    Table,
+    TableBody,
+    TableHeader
+} from '@patternfly/react-table';
+import { Messages } from '../../properties/Messages';
+import { assertNever, joinClasses, OuiaComponentProps } from '@redhat-cloud-services/insights-common-typescript';
+import { getOuiaProps } from '../../utils/getOuiaProps';
+import { Action, NotificationType, Notification } from '../../types/Notification';
+import { style } from 'typestyle';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Table/table';
+import { ActionComponent } from './ActionComponent';
+import { Button, ButtonVariant } from '@patternfly/react-core';
+import { GroupByEnum } from './Types';
+
+import './Table.scss';
+
+const pfBorderBottomClassName = style({
+    borderBottom: 'var(--pf-c-table--border-width--base) solid var(--pf-c-table--BorderColor)'
+});
+
+const noBorderBottom = style({
+    borderBottom: 'none !important'
```

Comment:
never fun to rely on !important, importing ah .scss directly into this component would likely eliminate the need for this

(or maybe not because yer build pipeline is different? 🤔 )

Reply:
The issue I encountered with this syntax is that PF was added *after* these kind of declarations, thus PF definition won here.
I suppose that some way to counter this is to check if we can bump the priority of typestyle.